tests/unit/python/fledge/common/test_configuration_cache.py

```diff
@@ -13,10 +13,13 @@
 @pytest.allure.story("common", "configuration_manager", "configuration_cache")
 class TestConfigurationCache:
 
-    def test_init(self):
-        cached_manager = ConfigurationCache()
+    @pytest.mark.parametrize("size", [
+        1, 10, 20, 1000
```

Comment:
Add unit test for 0, 1001 to ensure not only min/max API prevents them but functions are safe to that

Reply:
These tests are only meant for Configuration Cache and no access of min and max attributes that applies to configuration manager calls

Reply:
then at-least 0 test should be there;

And 0,1,1000,1001 should be around API